README.md

```diff
@@ -1,96 +1,112 @@
-# environment construction
+# High Dimensional Neural Network Potential(HDNNP)
 
-基本的にpythonのバージョン管理はpyenvを使うこと。  
-必要なコマンド(pyenv,pipenv,conda)のインストールについては省略する。
+> This is a implementation of High Dimensional Neural Network Potential(HDNNP) designed to reproduce Density Function Theory(DFT) calculation *effectivly* with high *flexibility*, *reactivity*.
 
-注意1：  
-2018/11/16時点で、Anaconda Cloud上に
+There is equivalent doc in Japanese [README.ja.md](https://github.com/ogura-edu/HDNNP/blob/master/README.ja.md).
 
-- ChainerMN
-- Chainer v5 (v5からChainerMNがマージされた)
+## Install
+
+Install this project by `git`.
 
-が存在しないため、anacondaを使う場合でもpipを使用してChainerMNをインストールする必要がある。  
-環境が壊れる可能性を承知した上で使用すること。  
-参考：http://onoz000.hatenablog.com/entry/2018/02/11/142347
+```shell
+git clone git@github.com:ogura-edu/HDNNP.git
 
-注意2：  
-2018/11/16時点で、pyenvを使ってanacondaをインストールしてある場合、pipenvによるインストールは失敗するという情報がある。  
-このバグの修正版は将来的にリリースされるらしい。  
-参考：https://github.com/pypa/pipenv/issues/3044
+# or if using ssh
+
+git clone git@github.com:ogura-edu/HDNNP.git
+```
 
-## pipenv (recommended)
+This project uses [Pipenv](https://github.com/pypa/pipenv) for development workflow. If you don`t have it, run this command to install.
 
-簡単かつ確実なインストール方法。
 
-環境変数`PIPENV_VENV_IN_PROJECT`を1に設定すると、  
-pipenvで作成されるpythonの仮想環境がこのプロジェクトの直下に作成される。(`/path/to/HDNNP/.venv/`)  
-以下のコマンドを実行するか、`~/.bashrc`に追記してプロセスを再起動することで変更が適用される。
+**macOS**
 ```
-export PIPENV_VENV_IN_PROJECT=1
+brew install pipenv
 ```
 
+## Setup
+### By Pipenv(Prefered)
+
+Same as by anaconda, but you need to install python rather than installing anaconda. This bug will be fixed in near future release.
```

Comment:
To be exact, it is a bug that `pipenv install` fails in environments already installed anaconda with pyenv. Since I have put the reference in `README.ja.md`, please also put here.